### PR TITLE
Adding new MF command

### PR DESCRIPTION
**Pull request by runleonarun**
## What are you changing in this pull request and why?

Adds new `mf validate-config` command. Originally added previously but removed [here](https://github.com/dbt-labs/docs.getdbt.com/pull/5887/files)

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/build/metricflow-commands.md [hunk 1]
```diff
@@ -26,13 +26,13 @@ Using MetricFlow with dbt Cloud means you won't need to manage versioning &mdash
...
 - You don't need to manage versioning &mdash; your dbt Cloud account will automatically manage the versioning for you.
-
+ 
 </TabItem>
 
 <TabItem value="cloud ide" label="dbt Cloud IDE">
 
 :::info
-You can create metrics using MetricFlow in the dbt Cloud IDE. However, support for running MetricFlow commands in the IDE will be available soon.
+You can create metrics using MetricFlow in the dbt Cloud IDE and [run `dbt sl validate`](/docs/build/validation#validations-command). Support for running more MetricFlow commands in the IDE will be available soon.
```
Comment (runleonarun) [inline website/docs/docs/build/metricflow-commands.md]:
```suggestion
You can create metrics using MetricFlow in the dbt Cloud IDE and run the [dbt sl validate](/docs/build/validation#validations-command) command. Support for running more MetricFlow commands in the IDE will be available soon.
```

File: website/docs/docs/build/metricflow-commands.md [hunk 2]
```diff
@@ -78,10 +78,9 @@ You can use the `dbt sl` prefix before the command name to execute them in the d
 - [`query`](#query) &mdash; Query metrics, saved queries, and dimensions you want to see in the command line interface. Refer to [query examples](#query-examples) to help you get started.
 - [`export`](#export) &mdash;  Runs exports for a singular saved query for testing and generating exports in your development environment. You can also use the `--select` flag to specify particular exports from a saved query.
 - [`export-all`](#export-all) &mdash; Runs exports for multiple saved queries at once, saving time and effort.
-
+- [`validate-configs`](#validate-configs) &mdash; Validates semantic model configurations.
```
Comment (WilliamDee) [inline website/docs/docs/build/metricflow-commands.md]: it should be just `validate` not `validate-configs`

File: website/docs/docs/build/metricflow-commands.md [hunk 3]
```diff
@@ -222,10 +221,8 @@ The list of available saved queries:
 
 The following command performs validations against the defined semantic model configurations.
 
-Note, in dbt Cloud you don't need to validate the Semantic Layer config separately. Running a dbt command (such as `dbt parse`, `dbt build`, `dbt compile`, `dbt run`) automatically checks it.
-
 ```bash
-
+dbt sl validate-configs # dbt Cloud users
```
Comment (WilliamDee) [inline website/docs/docs/build/metricflow-commands.md]: same thing here, should be just `dbt sl validate`, ``validate-config` is used only in the MF CLI, which would be `mf validate-config`, we are looking to rename that to be just `mf validate`, it just hasn't been done yet

File: website/docs/docs/build/validation.md [hunk 4]
```diff
@@ -14,11 +14,10 @@ The code that handles validation [can be found here](https://github.com/dbt-labs
 
 ## Validations command
 
-You can run validations against the defined semantic model configurations from the command line with the following [MetricFlow commands](/docs/build/metricflow-commands):
-
-Note, in dbt Cloud you don't need to validate the Semantic Layer config separately. Running a dbt command (such as `dbt parse`, `dbt build`, `dbt compile`, or `dbt run`) automatically checks it.
+You can run validations from dbt Cloud or the command line with the following [MetricFlow commands](/docs/build/metricflow-commands). In dbt Cloud, you need developer credentials to run `dbt sl validate-configs` in the IDE or CLI, and deployment credentials to run it in CI.
 
 ```bash
+dbt sl validate-configs # dbt Cloud users
```
Comment (WilliamDee) [inline website/docs/docs/build/validation.md]: ditto on the previous comments

File: website/docs/docs/build/metricflow-commands.md (hunk 2 above)
Comment (runleonarun) [inline website/docs/docs/build/metricflow-commands.md]:
```suggestion
- [`validate`](#validate) &mdash; Validates semantic model configurations.
```

File: website/docs/docs/build/metricflow-commands.md (hunk 3 above)
Comment (runleonarun) [inline website/docs/docs/build/metricflow-commands.md]:
```suggestion
dbt sl validate # dbt Cloud users
```

File: website/docs/docs/build/validation.md (hunk 4 above)
Comment (runleonarun) [inline website/docs/docs/build/validation.md]:
```suggestion
dbt sl validate # dbt Cloud users
```

File: website/docs/docs/build/metricflow-commands.md (hunk 2 above)
Comment (runleonarun) [reply website/docs/docs/build/metricflow-commands.md]: fixed in https://github.com/dbt-labs/docs.getdbt.com/pull/5930/commits/51a821ce13b69866bcf690b9f3761312725fef27

File: website/docs/docs/build/validation.md (hunk 4 above)
Comment (runleonarun) [reply website/docs/docs/build/validation.md]: Fixed in https://github.com/dbt-labs/docs.getdbt.com/pull/5930/commits/51a821ce13b69866bcf690b9f3761312725fef27

File: website/docs/docs/build/metricflow-commands.md (hunk 3 above)
Comment (runleonarun) [reply website/docs/docs/build/metricflow-commands.md]: fixed in https://github.com/dbt-labs/docs.getdbt.com/pull/5930/commits/51a821ce13b69866bcf690b9f3761312725fef27

**Comment by runleonarun** [comment]
@WilliamDee Fixed those spots and the link. Ready for re-review when you have a chance.

**Review by WilliamDee — APPROVED** [review]
lgtm!

**Review by nataliefiann — APPROVED** [review]
Hiya @runleonarun 

Great PR! Thanks for flagging this. No suggestions or recommendations from me. I've approved this for you. Please let me know if there's anything else I can do to help with this.

Kind Regards
Natalie

**Pull request merged**